Makefile

```diff
@@ -73,6 +73,14 @@ ifndef TANZU_CLI_TMC_UNSTABLE_URL
 TANZU_CLI_TMC_UNSTABLE_URL = ""
 endif
 
+ifndef TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL
+TANZU_CLI_E2E_TEST_CENTRAL_REPO_URL = gcr.io/eminent-nation-87317/tanzu-cli/test/v1/plugins/plugin-inventory:latest
+endif
+
+## update PATH with tanzu binary path, and tanzu CLI tools binaries path
+CURRENT_DIR = $(shell pwd)
+$(eval export PATH=$(CURRENT_DIR)/bin:$(CURRENT_DIR)/hack/tools/bin:$(PATH))
```

Comment:
I see that we are updating the path variable as part of the `Makefile`. What is the reason behind this? We should not update the PATH variable as part of the `Makefile`.

Can you explain the scenario where this is needed?

Reply:
We need to update PATH for tanzu binary and tools (from hack/toolgs/bin), i need these while running E2E tests, let me move under the e2e make target.

Reply:
removed in makefile and updated in github workflow